pages/common/yank.md

```diff
@@ -0,0 +1,23 @@
+# yank
+
+> Read input from stdin and display a selection interface that allows a field to be selected and copied to the clipboard.
+
+- Yank an environment variable key or value:
+
+`env | yank -d =`
+
+- Yank a field from a CSV file:
+
+`yank -d \", <file.csv`
+
+- Yank a whole line using the -l option:
```

Comment:
Again, this is verbatim from the docs. I don't think there is a need to mention 'using the -l' option. It is clear from the example that -l is used. I would just say - `Yank an entire line to clipboard:`